tests/plotting/test_plotting.py

```diff
@@ -1453,29 +1461,29 @@ def test_default_name_tracking():
 
 
 @skip_no_plotting
-def test_add_background_image_global():
+def test_add_background_image_global(sphere):
```

Comment:
What's the difference between using the global name vs. letting pytest magically pass it (I'm guessing via the `sphere` fixture in `conftest.py`)? I'm asking because there are plenty of other tests in this module that still use the global `sphere` name present in the module.

Reply:
I'm slowly trying to replace it.  Reasoning is that seems cleaner to use the fixture in `conftest.py`, and it's impossible to create side-effects as the mesh is copied each time it's passed in `pytest`.